HomeWorks/HW2/Bor/TireTests.cs

```diff
@@ -0,0 +1,85 @@
+// Модульные тесты Trie с использованием NUnit
+using NUnit.Framework;
+using TrieHomework;
+
+namespace TrieHomework.Tests;
+
+[TestFixture]
+public class TrieTests
+{
+    private Trie _trie;
+
+    [SetUp]
+    public void Setup()
+    {
+        _trie = new Trie();
+    }
+
+    [Test]
+    public void Add_NewElement_ReturnsTrue()
+    {
+        Assert.IsTrue(_trie.Add("test"));
```

Comment:
nit: В современном NUnit считаются предпочтительными тесты с `Assert.That`:
```suggestion
        Assert.That(_trie.Add("test"));
```